esignet-ida115-integration-impl/src/test/java/io/mosip/ida115/esignet/integration/service/IdaAuthenticatorImplTest.java

```diff
@@ -140,6 +223,8 @@ public void doKycAuth_withValidDetails_thenPass() throws Exception {
 		Mockito.when(restTemplate.exchange(Mockito.<RequestEntity<Void>>any(),
 				Mockito.<ParameterizedTypeReference<IdaResponseWrapper<IdaKycResponse>>>any()))
 				.thenReturn(responseEntity);
+		
+		Mockito.when(tokenIdManager.generateTokenId(Mockito.anyString(), Mockito.anyString())).thenReturn("1122334455");
```

Comment:
Why are we asserting the same value?
Its a positive test case, lets check for null/empty values in the kycAuthResult